lib/src/bt_device.dart

```diff
@@ -342,7 +328,9 @@ class GYWBtDevice with ChangeNotifier implements Comparable<GYWBtDevice> {
     }
   }
 
-  /// Turn the screen on
+  /// Turns the screen on and initialize it for future drawings.
```

Comment:
Keep the tense consistent.

```suggestion
  /// Turns the screen on and initializes it for future drawings.
```